api/src/main/java/org/xnio/XnioWorker.java

```diff
@@ -1424,5 +1429,71 @@ public int getQueueSize() {
             return -1;
         }
     }
+
+    /**
+     * {@link ExternalTaskPool} wrapper to prevent an delegate {@link TaskPool} from being shut down by the worker.
```

Comment:
This self-link isn't really needed.  Really the entire doc block isn't needed :)

Reply:
Fixed